src/shared/internal-calendar/internal-calendar.tsx

```diff
@@ -48,6 +51,9 @@ export const Component = (
         variant,
         between,
         type = "standalone",
+        showBorder,
```

Comment:
is no border meant to be the default? if so, then the `date-input` will need to specify the prop to true